data-raw/render_mexico_codes.R

```diff
@@ -1,13 +1,13 @@
 # mexico region codes
+# For ISO-3166-2 codes, see: https://www.iso.org/obp/ui/#iso:code:3166:MX
+# ISO codes were matched by name to INEGI codes in source data
+# Note that Level 1 codes are the first 2 characters of Level 2 codes
+# Level 2 codes remain as provided in source data
 mexico_codes <- vroom::vroom("data-raw/mexico_codes.csv")
 mexico_codes <- tibble::tibble(
-  level = c("level_1_region", "level_2_region"),
-  name = c("iso_3166_2", "inegi_code"),
-  codes = list(
-    dplyr::select(mexico_codes, region_level_1, iso_code) %>%
-      unique(),
-    mexico_codes
-  )
+  level = "level_1_region",
+  name = "iso_3166_2",
```

Comment:
this info is duplicated in Mexico as a field (or should be) and hence not needed here?

Reply:
agreed